### PR TITLE
Improve `webgl2_multisampled_renderbuffers` example.

### PR DESCRIPTION
**Pull request by WestLangley**
Somewhat subjective. Suggestions welcome.

### CODE REVIEW

File: examples/webgl2_multisampled_renderbuffers.html [hunk 1]
```diff
@@ -75,15 +82,13 @@
 
 				container = document.getElementById( 'container' );
 
-				camera = new THREE.PerspectiveCamera( 45, container.offsetWidth / container.offsetHeight, 1, 2000 );
+				camera = new THREE.PerspectiveCamera( 45, container.offsetWidth / container.offsetHeight, 10, 2000 );
```
Comment (WestLangley) [inline examples/webgl2_multisampled_renderbuffers.html:85]: Previous version has z-fighting issues, depending on the seed.

File: examples/webgl2_multisampled_renderbuffers.html [hunk 2]
```diff
@@ -95,10 +100,16 @@
 				group = new THREE.Group();
 
 				const geometry = new THREE.SphereGeometry( 10, 64, 40 );
-				const material = new THREE.MeshLambertMaterial( { color: 0xee0808 } );
+				const material = new THREE.MeshLambertMaterial( {
+					color: 0xee0808,
+					polygonOffset: true,
+					polygonOffsetFactor: 1, // positive value pushes polygon further away
+					polygonOffsetUnits: 1
```
Comment (WestLangley) [inline examples/webgl2_multisampled_renderbuffers.html:107]: This improves the render of the wireframe.

File: examples/webgl2_multisampled_renderbuffers.html [hunk 3]
```diff
@@ -95,10 +100,16 @@
...
+
+				} );
 				const material2 = new THREE.MeshBasicMaterial( { color: 0xffffff, wireframe: true } );
 
-				for ( let i = 0; i < 10; i ++ ) {
+				for ( let i = 0; i < 50; i ++ ) {
```
Comment (WestLangley) [inline examples/webgl2_multisampled_renderbuffers.html:112]: More to look at :-)

File: examples/webgl2_multisampled_renderbuffers.html [hunk 4]
```diff
@@ -149,6 +160,11 @@
 
 				//
 
+				const gui = new GUI();
+				gui.add( params, 'animate' );
```
Comment (WestLangley) [inline examples/webgl2_multisampled_renderbuffers.html:164]: Animation was obfuscating rendering issues. Allow animation to be disabled.

File: examples/webgl2_multisampled_renderbuffers.html [hunk 5]
```diff
@@ -172,7 +188,11 @@
 
 				const halfWidth = container.offsetWidth / 2;
 
-				group.rotation.y += clock.getDelta() * 0.1;
+				if ( params.animate ) {
+
+					group.rotation.y += 0.002;
```
Comment (WestLangley) [inline examples/webgl2_multisampled_renderbuffers.html:193]: Remove clock so a restarted animation continues where it left off -- and does not jump.

**Review by Mugen87 — APPROVED** [review]

**Pull request merged**